src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_format.py

```diff
@@ -0,0 +1,254 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import LongRunningOperation
+
+
+def _last_segment(resource_id):
+    return resource_id.split('/')[-1] if resource_id else None
+
+
+_units = [(1024 * 1024 * 1024 * 1024, 'TB'),
+          (1024 * 1024 * 1024, 'GB'),
+          (1024 * 1024, 'MB'),
+          (1024, 'kB'),
+          (1, 'B')]
+
+
+def _bytes_to_friendly_string(b):
+    '''
+    Formats the specified integer count of bytes as a friendly string
+    with units, e.g. 1024 -> "1kB"
+    '''
+
+    # Find the largest unit that evenly divides the input
+    unit = next(u for u in _units if (b % u[0]) == 0)
+
+    # Format the value with the chosen unit
+    return str((b // unit[0])) + unit[1]
+
+
+class LongRunningOperationResultTransform(LongRunningOperation):  # pylint: disable=too-few-public-methods
```

Comment:
You guys should consider adding this into CLI core. Or just fix this code so that it applies transform / LRO in the opposite order: https://github.com/azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/commands/__init__.py#L312